packages/@aws-cdk/aws-autoscaling/lib/auto-scaling-group.ts

```diff
@@ -234,6 +234,19 @@ export interface CommonAutoScalingGroupProps {
    * @default - Monitoring.DETAILED
    */
   readonly instanceMonitoring?: Monitoring;
+
+  /**
+   * Enable monitoring for group metrics.
+   *
+   * Group metrics describe the group rather than any of its instances (e.g GroupMaxSize, the group maximum size).
+   * You can enable monitoring for all group metrics using `GroupMetrics.ALL` or you can specify a subset of group metrics.
+   *
+   * Group metrics are reported in a granularity of 1 minute at no additional charge.
+   *
+   * You can also use the `emitAllGroupMetrics` and `emitGroupMetrics` methods
+   * @default - disabled
+   */
+  readonly groupMetricsCollections?: MetricsCollection[];
```

Comment:
Just `groupMetrics` is sufficient

```suggestion
  readonly groupMetrics?: MetricsCollection[];
```

Reply:
Question (and something I was thinking about) since this is a property on the constructor, should this resemble the CloudFormation property name: `MetricCollections`?